kotsadm/web/src/components/modals/UpdateCheckerModal.jsx

```diff
@@ -128,7 +128,7 @@ export default class UpdateCheckerModal extends React.Component {
         className="Modal SmallSize"
       >
         <div className="u-position--relative flex-column u-padding--20">
-          <span className="u-fontSize--largest u-fontWeight--bold u-color--tuna u-marginBottom--15">Configure automatic updates</span>
+          <span className="u-fontSize--largest u-fontWeight--bold u-color--tuna u-marginBottom--15">Automatic updates checks</span>
```

Comment:
I think this should say "Configure automatic update checks"

Reply:
What about something smaller, like "Configure checks"?

Reply:
If it's only on hover, then we can get away with more verbose description, if that's clearer.

Reply:
Oh wait, this is in the modal title itself. Yeah, let's go with verbose.